pkg/registries/registries.go

```diff
@@ -96,26 +105,47 @@ func mergedMirrorSets(icspRules []*apioperatorsv1alpha1.ImageContentSourcePolicy
 }
 
 // EditRegistriesConfig edits, IN PLACE, the /etc/containers/registries.conf configuration provided in config, to:
-// - Mark whole registries in insecureRegistries as insecure (TLS is not required, and TLS certificate verification is not required when TLS is used)
-// - Mark whole registries in blockedRegistries as blocked (any attempts to access them fail)
+// - Mark whole registries or wildcard entries in insecureRegistries as insecure (TLS is not required, and TLS certificate verification is not required when TLS is used)
+// - Mark whole registries or wildcard entries in blockedRegistries as blocked (any attempts to access them fail)
 // - Implement ImageContentSourcePolicy rules in icspRules.
 // “Whole registries” above means that the configuration applies to everything on that registry, including any possible separately-configured
 // namespaces/repositories within that registry.
+// "Wildcard entries" above means that we accept wildcards in the form of *.example.registry.com for insecure, blocked, and allowed registries only. We do not
+// accept them for mirror configuration.
```

Comment:
Allowed registries are not a parameter (they only show up in `policy.json`).

Reply:
Fixed